docs/content/toc.textile

```diff
@@ -16,7 +16,7 @@ h2. Data Ingestion
 * "Schema Design":../ingestion/schema-design.html
 * "Schema Changes":../ingestion/schema-changes.html
 * "Realtime Ingestion":../ingestion/realtime-ingestion.html
-* "Batch Ingestion":../ingestion/batch-ingestion.html
+* "Batch ingestion or Re-indexing or Delta ingestion":../ingestion/batch-ingestion.html
```

Comment:
you mean adding another file batch-delta-ingestion.md and link it in "Ingestion" section in the TOC file ? But that new file will only say what is updated in "docs/content/ingestion/batch-ingestion.md
" in this PR. is that enough or you are looking for something more?